RastenbergerAngularJS/src/main/webapp/app.js

```diff
@@ -0,0 +1,3 @@
+(function () {
```

Comment:
This is the main file for AngularJS code. The module "rastenbergerApp" is declared here and all needed modules are declared as dependencies. In this case "foodoffer-directives".